wandb/vendor/pynvml/pynvml.py

```diff
@@ -1,5 +1,16 @@
 #####
-# Copyright (c) 2011-2015, NVIDIA Corporation.  All rights reserved.
+#
+# Based on nvidia-ml-py version 11.515.48
+# Sourced from https://pypi.org/project/nvidia-ml-py/
+# Modifications:
+# - Added `NVML_DLL_PATH` env var check to the library initialization path
+# - Applied black formatting to improve readability
+# - Try different versions ("_v3", "_v2", "") of the functions:
```

Comment:
Older driver versions don't understand _v3's that is not the only option in the lib, so I had to add this enumeration/trial